test/builder/dummies.go

```diff
@@ -504,6 +513,20 @@ func DummyVirtualMachinePublishRequest(name, namespace, sourceName, itemName, cl
 	}
 }
 
+func VirtualMachineGroupPublishRequest(name, namespace string) *vmopv1.VirtualMachineGroupPublishRequest {
+	return &vmopv1.VirtualMachineGroupPublishRequest{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+		Spec: vmopv1.VirtualMachineGroupPublishRequestSpec{
+			Source:          DummyVMGroupName,
+			Target:          DummyContentLibraryName,
+			VirtualMachines: []string{DummyVirtualMachine0Name, DummyVirtualMachine1Name},
```

Comment:
Consider making this list of VM names as function param for better visibility and flexibility?